src/thunderbots/software/backend_output/grsim/motion_controller.h

```diff
@@ -0,0 +1,27 @@
+/**
+ * This file contains the content for the grSim motion controller
+ */
+
+#pragma once
+
+#include <utility>
+
+#include "util/constants.h"
+
+namespace MotionController
+{
+    /**
+     * Calculated robot velocities using Bang-bang motion controller for grSim
+     *
+     * @param robot The robot who's motion is to be controlled
+     * @param dest The destination of the robot
+     * @param desired_final_speed The target final speed of the robot
+     * @param desired_final_orientation The target final orientation of the robot
+     * @param delta_time The time that will be used to calculate the change in speed of
+     * the robot (based on constant acceleration)
```

Comment:
It's how much time has passed since the controller was last run (for that robot) (@EvanMorcom feel free to correct me here). Could probably use a more descriptive comment.

Reply:
@garethellis0 Matt is correct. Ideally delta_time  is length of time until the motion controller will be run again for each individual robot. Since I don't know how to predict that time, it is just the time since the controller was last run for each robot. It is used to calculated a change in speed using formula Vf = Vi + acceleration*delta_time. I can add a more descriptive comment.

Reply:
Improved delta_time description and added return comment